Add additional Miele fillingLevel sensors (#43370)
home-assistant/home-assistant.github.io: source/_integrations/miele.markdown

  - **Finish**: Shows the estimated date and time when the program will finish. If you've set a delayed start, it shows when the appliance is expected to complete the cycle, including the delay time.
  - **Plate**: Four to six sensors that show the current state of hob heating plates. The status mimics the display on the actual hob. For example, 0 is off, 5 is approximately 50% power, and "B" is power boost. Plates can only be monitored from Home Assistant, not controlled.
  - **TwinDos level**: Two sensors displaying the remaining level in the detergent containers in applicable washing machines. If the device does not support this sensor, the value is shown as `Unknown`.

>   - **Descaling, degreasing, milk pipework cleaning cycles counter**: A set of sensors displaying the total number of cycles that the appliance has run. These sensors will be available only for devices supporting these maintenance programs (such as coffee machines or ovens with steam addition).

Reviewer: ```suggestion
  - **Descaling, degreasing, milk pipework cleaning cycles counter**: A set of sensors displaying the total number of cycles that the appliance has run. These sensors are available only for devices that support these maintenance programs (such as coffee machines or ovens with steam addition).
```
